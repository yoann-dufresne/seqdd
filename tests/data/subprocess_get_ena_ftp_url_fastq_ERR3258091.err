  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   172  100   172    0     0   1360      0 --:--:-- --:--:-- --:--:--  1376
