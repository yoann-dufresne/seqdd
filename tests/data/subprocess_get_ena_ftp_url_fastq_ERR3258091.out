run_accession	fastq_ftp	fastq_md5	fastq_bytes
ERR3258091	ftp.sra.ebi.ac.uk/vol1/fastq/ERR325/001/ERR3258091/ERR3258091.fastq.gz	da03ef37d3f6f03c2a527449f7e562fd	3545446900
